### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getAttribute' of null

### ISSUE THREAD

**Issue by josephlyu** [dsrkafuu/aofuji-analytics#1]
Hi! Firstly thanks and congrats on this project!

I have followed the instructions stated in the documentation, and I believe that the Vercel deployment and MongoDB setup were carried out mostly correctly (the websites are stored correctly in the database).

However, it does not record the events and sessions. From the web inspector, it has this error: `Uncaught TypeError: Cannot read property 'getAttribute' of null`.

![image](https://user-images.githubusercontent.com/58096811/117557151-fac0ee00-b067-11eb-9f8d-d2d2454428a9.png)

![image](https://user-images.githubusercontent.com/58096811/117557162-1a581680-b068-11eb-8f84-d6c7fe0b2549.png)

![image](https://user-images.githubusercontent.com/58096811/117557178-322f9a80-b068-11eb-8df0-c1baa69b5ae4.png)

I have tried but didn't manage to understand the script, as I'm quite new to js. Do you know what the possible problem behind it is?

Many thanks!

**Comment by dsrkafuu** [comment]
There is nothing wrong with your setup.
The code generator in `Settings - Website` panel has not been updated to match the [latest tracker scripts](https://github.com/dsrkafuu/aofuji-tracker#cdn).

For example:

```js
<script
  async
  data-vaid="6a********************8c"
  data-vaapi="https://example.org/api"
  src="https://cdn.jsdelivr.net/npm/aofuji-tracker@0.1/lib/aofuji.min.js"
></script>
```

Replace `vaid` and `vaapi` with `aoid` and `aoapi`, and it should works.

I've also released `v0.1.3` which fixes this issue, updates the app and the code generator should work properly.

**Comment by josephlyu** [comment]
I updated to `v0.1.3` and it worked! Thank you!
